lib/src/base-styles/_base.scss

```diff
@@ -28,6 +28,7 @@
 @import 'nx-lists';
 @import 'nx-text-input';
 @import 'nx-tables';
+@import 'nx-tabs';
```

Comment:
Given that we aren't, currently, documenting (and thus supporting?) `nx-tabs` styles separately from their react component, I'd think they should be imported by the component implementation file not here in the base styles.